pkg/upgrade/control_plane.go

```diff
@@ -681,6 +688,24 @@ func (u *ControlPlaneUpgrader) updateAndUploadKubeadmKubernetesVersion() error {
 	return nil
 }
 
+func (u *ControlPlaneUpgrader) removeNodeFromKubeadmConfigMap(nodeName string) error {
+	original, err := u.targetKubernetesClient.CoreV1().ConfigMaps("kube-system").Get("kubeadm-config", metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrap(err, "error getting kubeadm configmap from target cluster")
+	}
+
+	updated, err := removeNodeFromClusterStatus(original, nodeName)
+	if err != nil {
+		return err
+	}
+
+	if _, err = u.targetKubernetesClient.CoreV1().ConfigMaps("kube-system").Update(updated); err != nil {
+		return errors.Wrap(err, "error updating kubeadm configmap")
```

Comment:
i would replace `configmap` with `ConfigMap` as per the sig-docs guide on API objects.